convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""
+    This program converts between US state names, and abbreviations. 
+    For example, converting "MN" to "Minnesota" 
+    or converting "California" to "CA".
+"""
+import json
 
-
-
-import json  # import json for json processing
 def main():
+    """
+    Read data from JSON file to start main method
+    """
     with open('data.json','r') as file:
         data = file.read()
         data = json.loads(data)
-
-    print(data)
     
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    state_abbreviations = data #dictionary of state abbreviations keys and state name values 
```

Comment:
you can avoid this assignment by fixing the data variable name issue above

Reply:
@claraj fixed sounds good, I resolved the above issue and removed this variable assignment.